### ISSUE TITLE
Upcoming Hard Fork at Block 4370000

### ISSUE THREAD

**Issue by sammy007** [sammy007/open-ethereum-pool#207]
In the next ETH HF block reward will be changed, the details https://github.com/5chdn/EIPs/blob/619b19cb85e42d6e18c2e0030c69d47765906e5b/EIPS/eip-649.md

I no longer support this pool development, so patches from "community" members welcome.
Maybe will add it myself later.

**Comment by sammy007** [comment]
The update is in `develop` branch https://github.com/sammy007/open-ethereum-pool/tree/develop

Disclaimer: I didn't test it properly, update changes block rewards when next HF will occur. It must work as expected cuz there are automated tests were performed. You are welcome to test it yourself and report issues.

**Comment by sammy007** [comment]
Block time in frontend and other stuff you must update yourself on your side.

**Comment by sammy007** [comment]
Don't forget to update to the latest Geth 1.7.1 or parity.

**Comment by trustfarm-dev** [comment]
@sammy007 Your Pool is widely used on alter chain of ETH. ETC/MUSIC/..
Your wallet is Private wallet then you can check Musicoin pool fee donation.
I've also donated some music to you. cheer up!
https://orbiter.musicoin.org/tx/0x8e56ed303886b062c21c320cd645ba55a6dd6be79304ec1ad8fd40654b6ea5a1

**Comment by Raja4Shekar** [comment]
@sammy007 kindly post the procedure to update the pool. I have just now started using it. Thank you.

**Comment by sammy007** [comment]
```
git checkout origin/develop -b develop
make clean && make
```

**Comment by sammy007** [comment]
@trustfarm-dev many thanks.

**Comment by Raja4Shekar** [comment]
@sammy007 Thank you
I have also used 'extra data' as mentioned in another post. But I hope it will  take effect only when found a block. My open ethereum pool is at www.miningcryptopools.com and i hope i have done things appropriately. But kindly let me know if I have to consider any points which I need to implement like 'extra data'

I think currently most of the open ethereum pool owner are using 0% fee. Once the pool is used by many users then, the fees may increased to 1% so that you will also get the 10% fee share. You need to cheer up. Surely most of the users will need your support on this. It is that it will take time to find a block for them and once things have got settled down, you will also start getting the fee. So please continue your support. Thank you once again.

**Comment by Raja4Shekar** [comment]
@sammy007 i have found the first block https://etherscan.io/block/4355924#comments 
Will also send one ether to you by this evening

Thank you a lot
Cheer up

**Comment by Raja4Shekar** [comment]
@sammy007 found out it to be an uncle rather than a Block :( is there anything that I can do to improve the chance of getting a full Block rather an Uncle.

**Comment by Raja4Shekar** [comment]
@sammy007 Got excited got an another block 👍 
Will send you 1 ether in total by this evening

**Comment by Raja4Shekar** [comment]
@sammy007 sent one Ether as said! https://etherscan.io/tx/0x14401192031ebb65657ecb98c7d8d342f4954c3ed3cba2c6fe5671851494ff03

Will increase the pool fee to 1% once I get users, so that you will get 10% fee share.
Thank you
Cheer up

**Comment by sammy007** [comment]
@Raja4Shekar Cool. I highly advice to use parity if you are still using geth. It's much faster, regarding uncle rate I have no idea tbh, try to set block refresh interval to 500ms. Miner latency also matters.

**Comment by Raja4Shekar** [comment]
@sammy007 
- I am now using the `develop` branch (hope things will go smooth). 
- Also using geth instance `Geth/v1.7.1-stable-05101641/linux-amd64/go1.9` but will update to parity sooner 
- Also, changed the block refresh interval to 500ms (will use it in the next round)

Thank you for the suggestions.

**Comment by sammy007** [comment]
Looks like geth is more preferable now, there is rpc regression in parity which can cause unlocker to mark blocks falsely as orphans. Geth released critical update (again). Pretty hard to follow this shitcoin due to its retarded development process.

**Comment by Raja4Shekar** [comment]
Ok, I will continue to use geth. Will try and update geth some time later.

**Comment by icemining** [comment]
Thank's sammy007, you got also 1 Ether from icemining.io for your help.

BTW, Pool-Fee will be 0% for next time!

**Comment by alptolga** [comment]
@Raja4Shekar 
Are you mining ethereum? What is your hashrate?

**Comment by Raja4Shekar** [comment]
@alptolga 
Yes, I am mining ethereum and my hashrate currently is about 30 to 33 GH/s

Currently the pool miningcryptopools.com is in testing phase, need to work on payments which I have planned to do after 2 weeks.

**Comment by alptolga** [comment]
@Raja4Shekar 
payment process is not so hard to make. I am mining etc. It is impossible to find block with 1.5 gh on ethereum. Good luck for ethereum
